tests/sles4sap/forkbomb.pm

```diff
@@ -27,6 +28,35 @@ sub run {
     my $sid = get_required_var('INSTANCE_SID');
     my $instance_id = get_required_var('INSTANCE_ID');
     my $sapadm = $self->set_sap_info($sid, $instance_id);
+
+    my $package_version = script_output "rpm -q --qf '%{VERSION}' saptune";
+    record_info('saptune version', $package_version);
+    my $note_id = '2578899';
+    my $conf_kernel_pid_max_old = script_output("grep kernel.pid_max /usr/share/saptune/notes/$note_id", proceed_on_failure => 1);
+    record_info('Old conf value', $conf_kernel_pid_max_old);
+    my $os_kernel_pid_max_old = script_output('sysctl -a | grep kernel.pid_max', proceed_on_failure => 1);
```

Comment:
This could've been:
```
my $os_kernel_pid_max_old = script_output('sysctl kernel.pid_max', proceed_on_failure => 1);
```
No need to list all sysctl parameters and grep.

Reply:
Thanks.  I will enhance it in the future.